client/components/voip/composer/NotAvailableOnCall/NotAvailable.stories.tsx

```diff
@@ -0,0 +1,16 @@
+import { Story } from '@storybook/react';
+import React from 'react';
+
+import NotAvailableContent from './NotAvailableContent';
+import NotAvailableContentWrapper from './NotAvailableContentWrapper';
+
+export const Default: Story = () => (
+	<NotAvailableContentWrapper>
+		<NotAvailableContent text='Messages are not available on phone calls' icon='message' />
+	</NotAvailableContentWrapper>
+);
+
+export default {
+	title: 'components/Composer/NotAvailableOnCall',
+	component: Default,
```

Comment:
It should be `NotAvailableOnCall` here.

Reply:
this component has useTranslation on it...

Reply:
It's fine, it uses a mocked version of `TranslationProvider`.